### PR TITLE
LPS-188085 Remove LPS-155692 FeatureFlag and add new portal property

### PR DESCRIPTION
**Pull request by gislayne-vitorino**

### CODE REVIEW

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Comment by gislayne-vitorino** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/searchportlet/SearchFacets.testcase [hunk 1]
```diff
@@ -780,7 +780,7 @@ definition {
 
 	@priority = 4
 	test AssertNonAdminsCannotAddCertainWidgets {
-		property custom.properties = "feature.flag.LPS-155692=true";
+		property custom.properties = "layout.user.plid.access.denied=true";
```
Comment (pei-jung) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/searchportlet/SearchFacets.testcase:783]: Shouldn't we set the property to false here? Enabling the FF means allowing user pages to be accessed by plid.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/upgrades/PortalConfigurationUpgrade.testcase [hunk 2]
```diff
@@ -1,7 +1,7 @@
 @component-name = "portal-upgrades"
 definition {
 
-	property custom.properties = "feature.flag.LPS-155692=true";
+	property custom.properties = "layout.user.plid.access.denied=true";
```
Comment (pei-jung) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/upgrades/PortalConfigurationUpgrade.testcase:4]: Same here.

**Review by pei-jung — CHANGES_REQUESTED** [review]

**Review by pei-jung — APPROVED** [review]
ci:forward

**Comment by pei-jung** [comment]
ci:forward

**Comment by pei-jung** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
ci:stop

**Comment by pei-jung** [comment]
@patriciajeanperez when you get a chance, can you check if the failures are related?

**Comment by pei-jung** [comment]
ci:forward:force

**Comment by pei-jung** [comment]
ci:forward:force

**Pull request closed without merging**